webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -127,7 +128,15 @@ export class SearchableChannelList extends React.PureComponent<Props, State> {
         const ariaLabel = `${channel.display_name}, ${channel.purpose}`.toLowerCase();
         let channelTypeIcon;
 
-        if (isArchivedChannel(channel)) {
+        if (channel.shared) {
```

Comment:
Can a shared channel be archived? (Or does achieving a channel remove its shared status?)

Reply:
Yep...I moved this up. So that "archive" takes presedence.